src/scripts/domParsers/videoUtils.ts

```diff
@@ -6,7 +6,8 @@ export module VideoUtils {
 
 	export enum SupportedVideoDomains {
```

Comment:
Currently, all of our domain-specific video-grabbing logic lives in videoUtils, and we use subtly different ways to extract videos from each differing domain, but the end purpose is still the same. I wonder if we can break this up further into smaller classes to make things a bit simpler.